packages/cardpay-reward-programs/tests/rules/test_staking.py

```diff
@@ -98,7 +98,7 @@ def run_query(self, table_query, vars, aux_table_query):
     return rule
 
 
-def get_amount(result, payee, idx):
+def get_amount(result, payee):
```

Comment:
idx was never used here, so I've removed it and fixed the calls in the rest of the tests.